<ui version="4.0" >
 <class>ossimPlanetQtNewHttpConnection</class>
 <widget class="QDialog" name="ossimPlanetQtNewHttpConnection" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Connection Information</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="theProxyPassword" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Password for your HTTP proxy (optional)</string>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel1_3" >
        <property name="text" >
         <string>Proxy Host</string>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="buddy" >
         <cstring>theProxyHost</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="TextLabel1_3_2" >
        <property name="text" >
         <string>Proxy Port</string>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="buddy" >
         <cstring>theProxyPort</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="theName" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Name of the new connection</string>
        </property>
        <property name="frame" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="theProxyHost" >
        <property name="toolTip" >
         <string>Name of your HTTP proxy (optional)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TextLabel1" >
        <property name="text" >
         <string>URL</string>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="buddy" >
         <cstring>theUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="theProxyUser" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Your user name for the HTTP proxy (optional)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="theProxyPort" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Port number of your HTTP proxy (optional)</string>
        </property>
        <property name="maxLength" >
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="TextLabel1_3_3" >
        <property name="text" >
         <string>Proxy User</string>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="buddy" >
         <cstring>theProxyUser</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel1_2" >
        <property name="text" >
         <string>Name</string>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="buddy" >
         <cstring>theName</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="TextLabel1_3_4" >
        <property name="text" >
         <string>Proxy Password</string>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="buddy" >
         <cstring>theProxyPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="theUrl" >
        <property name="toolTip" >
         <string>HTTP address of the Web Map Server</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="theOkButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="theCancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>theName</tabstop>
  <tabstop>theUrl</tabstop>
  <tabstop>theProxyHost</tabstop>
  <tabstop>theProxyPort</tabstop>
  <tabstop>theProxyUser</tabstop>
  <tabstop>theProxyPassword</tabstop>
  <tabstop>theOkButton</tabstop>
  <tabstop>theCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
